diablo/jobs/util.py

```diff
@@ -223,13 +225,15 @@ def schedule_recordings(all_approvals, course):
             Latest approved_by_uid: {approval.approved_by_uid}
         """)
 
+    return scheduled
+
 
 def _adjust_time(military_time, offset_minutes):
     hour_and_minutes = military_time.split(':')
-    return datetime.combine(
-        date.today(),
-        time(int(hour_and_minutes[0]), int(hour_and_minutes[1])),
-    ) + timedelta(minutes=offset_minutes)
+    hour = int(hour_and_minutes[0])
+    minutes = int(hour_and_minutes[1])
+    timezone = pytz.timezone(app.config['TIMEZONE'])
+    return datetime.combine(date.today(), time(hour, minutes), tzinfo=timezone) + timedelta(minutes=offset_minutes)
```

Comment:
I included timezone fix in this PR. https://jira.ets.berkeley.edu/jira/browse/DIABLO-297